core/templates/dev/head/components/CkEditorWidgets.js

```diff
@@ -0,0 +1,190 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Rename the file to `CkEditorWidgetsGenerator.js`

Reply:
ok

Reply:
done

Reply:
done